src/common/utils.tsx

```diff
@@ -385,3 +385,7 @@ export const ftDecimals = (value: number | string | BigNumber, decimals: number)
 export const ftUnshiftDecimals = (value: number | string | BigNumber, decimals: number) => {
   return initBigNumber(value).shiftedBy(decimals).toString(10);
 };
+
+export function getLocaleDecimalSeparator() {
+  return Intl.NumberFormat().formatToParts(1.1).find(part => part.type === 'decimal')?.value
```

Comment:
Is this function guaranteed to _only_ run/render on the client? My understanding is that all or any of the explorer pages can be rendered on the server via nextjs, and in those cases, this will use whatever the default nodejs server locale is (something like en-US iirc).

/cc @kyranjamie @He1DAr

Reply:
@zone117x  Indeed, it seems this function is called on server side as well, before being called on client side.
This has no effect since the correct locale is used on client side. I have tested multiple languages and it works as expected.
We could return a default value or return an error when called from server side if you prefer.